testcode/assembler/Makefile

```diff
@@ -1,8 +1,13 @@
 
```

Comment:
No, I haven't. I think moving it there would be wise after the last known inconsistency to the spec, the auto-detection of long branches, is taken care of.

It would be also wise, to integrate such a test for every cpu the assembler is capable of, not just the 4510.